src/app/pages/storage/components/dashboard-pool/topology-card/topology-card.component.spec.ts

```diff
@@ -85,7 +85,7 @@ describe('TopologyCardComponent', () => {
 
     // Redundancy level should match data VDEVs
     expect(captions[1]).toHaveText('Metadata');
-    expect(values[1]).toHaveText('Redundancy Mismatch');
+    expect(values[1]).toHaveText('1 x MIRROR | 3 wide | 8 TiB');
```

Comment:
For this test let's have both redundancy mismatched and matched scenarios. Both Metadata and Dedup need to match redundancy level of data vdevs. Perhaps leave the metadata vdevs as you already have it but add dedup topology category to mockstorage and deliberately mismatch it? That way we can test for the error message.